db/migrate/20220107065750_add_feature_restrictions_to_assignment.rb

```diff
@@ -0,0 +1,7 @@
+class AddFeatureRestrictionsToAssignment < ActiveRecord::Migration[6.1]
+  disable_ddl_transaction!
+  def change
+    add_column :assignments, :feature_restrictions, :jsonb, default: {}
+    add_index  :assignments, :feature_restrictions, using: :gin, algorithm: :concurrently
```

Comment:
We don't need index for this as per our usage patterns

Reply:
I will remove it sir.